apache/map.jinja

```diff
@@ -186,3 +186,6 @@
 {# merge the os family/codename mod_sec's specific data over the defaults #}
 {% do apache.update({ 'mod_security': modsecurity }) %}
 
+{# check if user wants formula to skip 'apache service manipulation' #}
```

Comment:
Since we're only using Pillar and no OS-dependent lookup, there is no need to change `map.jinja` 
Please remove the changes here.